tests/AttributeTests/DomainFromConfigAttributeTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Spatie\RouteAttributes\Tests\AttributeTests;
+
+use Illuminate\Support\Facades\Config;
+use Spatie\RouteAttributes\Tests\TestCase;
+use Spatie\RouteAttributes\Tests\TestClasses\Controllers\DomainFromConfigTestController;
+
+class DomainFromConfigAttributeTest extends TestCase
+{
+    /** @test */
+    public function it_can_apply_a_domain_on_the_url_of_every_method()
+    {
+        Config::set('domains.test', 'config.localhost');
```

Comment:
use the helper `config` function instead of the facade.

Reply:
good spot - will correct this.